README.md

```diff
@@ -1 +1,47 @@
-# Riyaz-Portfolio
\ No newline at end of file
+# Riyaz-Portfolio
+
+## Built With
+
+HTML
+CSS
+
+#### Getting Started
+
+https://r-ahmed2022.github.io/Riyaz-Portfolio/
+
+#### Getting Started
+
+Instructions on how to access my project locally. To Access my project locally please follow the following instructions;
+
+Enter this url: https://github.com/r-ahmed2022/Riyaz-Portfolio in your web browser.
+Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+Select download Zip option from drop down menu.
```

Comment:
[OPTIONAL] I recommend you remove one of these `Getting Started` headers, they make your documentation look a little weird. Additionally, you may want to use this markup for your instructions:
```
git clone git@github.com:r-ahmed2022/Riyaz-Portfolio.git
```
If this looks cool to you and you want to use it for this and future README, all you have to do is wrap whatever text you want to bash inside three ticks (```). 😉